app/strategies/index.js

```diff
@@ -0,0 +1,20 @@
+const STRATEGIES = {
+    const: require('./const'),
+    fibonacci: require('./fibonacci'),
+    linear: require('./linear'),
+    powerOf2: require('./powerOf2'),
+    random: require('./random'),
+};
+const DEFAULT_STRATEGY = 'const';
```

Comment:
why const? AFAIK fibonacci is the best strategy in terms of self-DDOS

Reply:
Just because it was used before strategies introduction. Could be changed to any existing.